.eslintrc.js

```diff
@@ -35,6 +34,7 @@ module.exports = {
     "semi": "off",  // required for a proper work
     "@typescript-eslint/semi": ["error", "always"],
     "@typescript-eslint/strict-boolean-expressions": "off",
-    "sonarjs/cognitive-complexity": ["error", 16]
+    "sonarjs/cognitive-complexity": ["error", 16],
+    "sonarjs/no-identical-functions": 0
```

Comment:
Why are we turning this rule off?

Reply:
Because the deprecated method is same code with new method, just name is different. Should i do it inline instead of config this rule in .eslintrc, in this case?

Reply:
yes, we should do it inline (we don't need to turn this off for whole package). but i don't follow why we need to turn this off, as i don't think we're repeating code here.

Reply:
The warning makes sense - you should not copy the whole function. Instead, you should call the new function from the old one, as I wrote in the review comment: https://github.com/scale-it/algo-builder/pull/539/files#r770378696